projects/com.oracle.truffle.llvm.parser.base/src/com/oracle/truffle/llvm/parser/base/util/LLVMBitcodeTypeHelper.java

```diff
@@ -380,18 +382,30 @@ public static LLVMLogicalInstructionType toLogicalInstructionType(BinaryOperator
         return llvmtypes;
     }
 
+    /* (non-Javadoc)
```

Comment:
Can you remove these comments? I do not think that they make the code more readable. They are probably auto-generated?